Surface_mesh_skeletonization/include/CGAL/Mean_curvature_flow_skeletonization.h

```diff
@@ -375,14 +372,14 @@ std::size_t collapse_short_edges();
   Mean_curvature_flow_skeletonization(const TriangleMesh& tmesh,
                                       VertexPointMap vertex_point_map,
                                       const Traits& traits = Traits())
-    : m_traits(traits), m_weight_calculator(m_tmesh)
+    : m_traits(traits), m_weight_calculator(true)
```

Comment:
```suggestion
    : m_traits(traits), m_weight_calculator(true /*use_secure_version*/)
```
?

Reply:
But true, adding a comment is a good idea.